src/bootstrap/configure.py

```diff
@@ -174,36 +174,36 @@ def v(*args):
 
 
 def p(msg):
-    print("configure: " + msg)
+    print("configure: {}".format(msg))
 
 
 def err(msg):
-    print("configure: error: " + msg)
+    print("configure: error: {}".format(msg))
```

Comment:
I don't really see how this is an improvement.

Reply:
Lines such as 438 also used to look like this
```return "'" + value + "'"```
In this case string concatenation is both slower and not very readable compared to str.format that is why I changed it, and I also changed the other places where string concatenation was used. 
The best would of course be to use fstring but since it came out with python 3.6 I did not want to introduce any version dependency.

Reply:
Should I remove that specific part?